email-common/src/main/java/io/commercetools/sunrise/email/MessageEditor.java

```diff
@@ -0,0 +1,21 @@
+package io.commercetools.sunrise.email;
+
+import javax.annotation.Nonnull;
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+/**
+ * A functional interface for modifying a message without needing to handle checked exceptions like the
+ * {@link MessagingException}s thrown by {@link MimeMessage}.
+ */
+@FunctionalInterface
+public interface MessageEditor {
```

Comment:
It is worse, it has multiple send methods: http://commons.apache.org/proper/commons-email/javadocs/api-1.3.3/org/apache/commons/mail/Email.html